src/form/radiology-orders/add-radiology-order/radiology-order-form.component.tsx

```diff
@@ -205,6 +206,11 @@ export function RadiologyOrderForm({
         id="radiologyOrderForm"
       >
         <div className={styles.form}>
+          <ExtensionSlot
+            name="top-of-lab-order-form-slot"
```

Comment:
```suggestion
            name="top-of-imaging-order-form-slot"
```

We might also want to change this repo to `esm-imaging-app`

Reply:
so, what do you advise?

Reply:
renaming the app to imaging

Reply:
Okay, how about creating the slot from esm-billing-app

Reply:
> We might also want to change this repo to `esm-imaging-app`

Sorted